### PR TITLE
Add changes to ynion branch

### PR DESCRIPTION
**Pull request by ynion-mabeamae**
Please review my code. Thank you!

### CODE REVIEW

File: ynion.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        choice = int(input("Enter your choice: "))
+
+        match choice:
+            case 1:
+                print("\nDate of Birth : July 29, 2004")
+                print("Age : 20")
+                print("Email : mabeamaeynion@gmail.com")
+                print("Religion : Roman Cathokic")
+            case 2:
+                print("\nGoals: To learn more about programming and improve my problem-solving skillls.")
```
Comment (citron-kathleen) [inline ynion.py]: Please check your code

File: ynion.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        print("4. Exit")
+        
+        choice = int(input("Enter your choice: "))
+
+        match choice:
+            case 1:
+                print("\nDate of Birth : July 29, 2004")
+                print("Age : 20")
+                print("Email : mabeamaeynion@gmail.com")
+                print("Religion : Roman Cathokic")
```
Comment (delima-justine) [inline ynion.py]: It should be "Roman Catholic".

**Review by citron-kathleen — CHANGES_REQUESTED** [review]
Please check your spelling.

File: ynion.py (hunk 1 above)
Comment (delima-justine) [inline ynion.py]: Remove extra 'l' in "skills"

File: ynion.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            case 2:
+                print("\nGoals: To learn more about programming and improve my problem-solving skillls.")
+            case 3:
+                print(f"\nComments")
```
Comment (relente-patriciajoy) [inline ynion.py]: Remove f-string

File: ynion.py (hunk 2 above)
Comment (quiambao-mapatriciaanne) [inline ynion.py]: Please fix the word "Cathokic" to "Catholic"

**Review by relente-patriciajoy — CHANGES_REQUESTED** [review]
Please check the typos before I approve.

**Review by quiambao-mapatriciaanne — CHANGES_REQUESTED** [review]
Please fix your spelling first before I approve it.

**Review by delima-justine — CHANGES_REQUESTED** [review]
Please review the code again based on the recommendations before I approve. Thank you!

**Review by relente-patriciajoy — APPROVED** [review]
Great job!

**Review by delima-justine — APPROVED** [review]
Great!

**Review by quiambao-mapatriciaanne — APPROVED** [review]
Well done!

**Review by citron-kathleen — APPROVED** [review]
Good job!

**Pull request merged**